src/components/Admin/GameLoader.jsx

```diff
@@ -0,0 +1,136 @@
+import { useTranslation } from "react-i18next";
+import "@/i18n/i18n";
+import { handleCsvFile, handleJsonFile } from "@/lib/utils";
+
+function isValidFileType(file, allowedTypes) {
+  const fileName = file.name.toLowerCase();
+  const fileExtension = fileName.split(".").pop();
+
+  if (!allowedTypes[fileExtension]) {
+    return false;
+  }
+
+  const mimePattern = allowedTypes[fileExtension].pattern;
+  return mimePattern.test(file.type);
+}
+
+const GameLoader = ({
+  gameSelector,
+  send,
+  setError,
+  setCsvFileUpload,
+  setCsvFileUploadText
+}) => {
+  const { i18n, t } = useTranslation();
+  return (
+    <div className="flex flex-col rounded  border-2">
+      <div className="flex translate-y-3 flex-row  items-center justify-center space-x-5 p-2">
+        {gameSelector.length > 0 ? (
+          <div>
+            <select
+              id="gameSelector"
+              defaultValue={""}
+              className="rounded border-2 bg-secondary-500 text-foreground"
+              onChange={(e) => {
+                send({
+                  action: "load_game",
+                  file: e.target.value,
+                  lang: i18n.language,
+                });
+              }}
+            >
+              <option disabled value="">
+                {t("Select question set")}
+              </option>
+              {gameSelector.map((value, index) => (
+                <option key={`set-${index}`} value={value}>
+                  {value.replace(".json", "")}
+                </option>
+              ))}
+            </select>
+          </div>
+        ) : null}
+        <div id="gamePickerFileUploadButton">
+          <label htmlFor="gamePickerFileUpload">
+            <div className="flex flow-row space-x-2 items-center">
```

Comment:
```suggestion
            <div className="flow-row flex items-center space-x-2">
```
eslint tailwind classname order